pip/rules.bzl

```diff
@@ -288,18 +296,9 @@ deploy_pip = rule(
     attrs = {
         "target": attr.label(
             mandatory = True,
-            allow_single_file = [".tar.gz"],
             providers = [PyDeploymentInfo],
             doc = "`assemble_pip` label to be included in the package",
         ),
-        "snapshot": attr.string(
-            mandatory = True,
-            doc = "Remote repository to deploy pip snapshot to"
-        ),
-        "release": attr.string(
-            mandatory = True,
-            doc = "Remote repository to deploy pip release to"
```

Comment:
because the initial logic of this rule is uploading with "release url", "username", and "password". With aws CA we use "~/.pypirc" instead. And that's one of the reason that the open source repo might not accept our PR. Not all of the companies use CA and I didn't want spend too much time on it to make it more compatible